src/platform/Darwin/DnssdContexts.cpp

```diff
@@ -368,7 +362,12 @@ void BrowseContext::DispatchSuccess()
 void BrowseContext::DispatchPartialSuccess()
 {
     sContextDispatchingSuccess = this;
-    callback(context, services.data(), services.size(), false, CHIP_NO_ERROR);
+    std::vector<DnssdService> dnsServices;
```

Comment:
once this merges #32740, i need to rebase this. Right now I am keeping the changes here as well.